web/js/modules/measure/reducers.js

```diff
@@ -1,21 +1,40 @@
-import { assign as lodashAssign } from 'lodash';
-import { CHANGE_UNITS, TOGGLE_MEASURE_ACTIVE } from './constants';
+import {
+  CHANGE_UNITS,
+  UPDATE_MEASUREMENTS,
+  TOGGLE_MEASURE_ACTIVE,
+} from './constants';
 
 const defaultState = {
   isActive: false,
   unitOfMeasure: 'km',
+  allMeasurements: {
+    'EPSG:3413': {},
```

Comment:
Suggestion: pull the spacial reference from the config so it could work with `EPSG:3857` as well (on non-production versions of Worldview).